crates/types/src/node_id.rs

```diff
@@ -273,6 +273,10 @@ impl PlainNodeId {
         PlainNodeId(id)
     }
 
+    pub fn is_valid(self) -> bool {
+        self.0 != 0
+    }
+
```

Comment:
I guess these changes in this file are unrelated.

Reply:
Yes. Those are for future changes.